client-v2/src/main/java/com/clickhouse/client/api/query/GenericRecord.java

```diff
@@ -500,4 +496,17 @@ public interface GenericRecord {
     ClickHouseBitmap getClickHouseBitmap(String colName);
 
     ClickHouseBitmap getClickHouseBitmap(int index);
+
+    TableSchema getSchema();
+
+    /**
+     * Returns all values of this record in Map.
+     *
+     * @return a Map of column names and values.
+     */
+    default Map<String, Object> getAll() {

```

Comment:
lets rename it to `getValues` or `asMap`. 
I would prefer having no default implementation in interfaces because `default` was designed to not break things when new stream API was introduced in java. I think, that only this case is valid usage of `default`.